doc/design/web.md

```diff
@@ -0,0 +1,153 @@
+# Web Interface design
+
+This design doc will talk about features and web pages needed to let users manage cloud paddle jobs.
+
+## Feature List
+
+- Account Management
+    - Registration, send email to inform if registration succeeded
+    - Account Login/Logout
+    - Password changing, find back
+    - Download SSL keys
+- Jupiter Notebook
+    - Private Jupiter Notebook environment to run Python scripts
+    - Private workspace
+    - Submit job from Jupiter Notebook
+- Job Dashboard
+    - Job history and currently running jobs
+    - Performance Monitoring
+    - Quota Monitoring
+- Datasets
+    - Public Dataset viewing
+    - Upload/Download private datasets
+    - Share datasets
+- Models
+    - Upload/Download models file
+    - Share/Publish Models
+- Paddle Board
+    - Training metrics visualization
+        - cost
+        - evaluator
+        - user-defined metrics
+- Serving
+    - Submit serving instances
+    - Deactivate serving
+    - Serving performance monitoring
+
+## Account Management
+
+Account management page is designed to satisfy multi-tenant use cases. One account should have a unique account ID for login, and this account owns one access key to one unique [Kubernetes namespace](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/) cluster. Multiple users can log in to this account ID and operate jobs and data files. The only "master user" can do modifications like increase quota or manage account settings.
```

Comment:
"log in to this account ID" or log in to its own account which belongs to the group. If so, "master user" could be "group owner".